install.sh

```diff
@@ -53,8 +55,10 @@ if [  -z "$CLIENT_ID" ] || [ -z "$CLIENT_SECRET" ]; then
 fi
 
 setup_nextcloud &
+setup_nextcloud_pid=$!
 build_app &
-wait
+wait %2 || exit 1
```

Comment:
this will actually fail the build script when something fails in the background  jobs